web/client/plugins/Map.jsx

```diff
@@ -286,5 +286,6 @@ const selector = createSelector(
 );
 module.exports = {
     MapPlugin: connect(selector)(MapPlugin),
-    reducers: { draw: require('../reducers/draw') }
+    reducers: { draw: require('../reducers/draw') },
+    epics: { addGeometryFieldLayer, addGeometryFieldConfig}
```

Comment:
probably this side effect have to be added if the featureGrid is present @mbarto what do you think?